src/schedule/schedule.styles.tsx

```diff
@@ -0,0 +1,49 @@
+import styled, { css } from "styled-components";
+import { MIN_COLUMN_WIDTH } from "./const";
+import { ErrorDisplay } from "../error-display";
+import { Border, Colour, Radius, Spacing } from "../theme";
+
+// =============================================================================
+// STYLING
+// =============================================================================
+interface ScheduleProps {
+    $loading: boolean | undefined;
+}
+
+export const Container = styled.div<ScheduleProps>`
+    min-width: 320px;
+    max-width: 90vw;
+    box-sizing: border-box;
+    ${(props) => {
+        if (props.$loading) {
+            return css`
+                :hover {
```

Comment:
missing prefix, in styled components v6 this is mandatory

```suggestion
                &:hover {
```